CHANGELOG.next.asciidoc

```diff
@@ -1022,6 +1022,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add dns.question.top_level_domain fields for sysmon DNS events. {pull}23046[23046]
 - Add support for sysmon v13 events 24 and 25. {issue}24217[24217] {pull}24945[24945]
 - Changed the log level of the "Successfully published events" message from `info` to `debug` to reduce verbosity of the `info` logging level. To track event log reader activity use the `published_events` metric. {pull}25617[25617]
+- Adding field and value in the error/warning message that is generated when there is an issue with the equals condition. {issue}25481[25481] {pull}25530[25530] 
```

Comment:
This changes libbeat so the message must go in the section affects all beats section.